androidApp/src/main/java/com/ajay/seenu/expensetracker/android/domain/util/Utils.kt

```diff
@@ -0,0 +1,55 @@
+package com.ajay.seenu.expensetracker.android.domain.util
+
+import android.content.Context
+import android.graphics.Bitmap
+import android.net.Uri
+import android.provider.OpenableColumns
+import androidx.core.net.toUri
+import java.io.File
+import java.io.FileOutputStream
+
+fun saveBitmapToFile(context: Context, bitmap: Bitmap): Uri {
+    val file = File(context.cacheDir, "${System.currentTimeMillis()}.jpg")
```

Comment:
In order to prevent race condition, can we add some prefix like file id before the `System.currentTImeMillis` before na ?

Reply:
Yes. Added UUID to file name.